modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/mob.dm

```diff
@@ -0,0 +1,24 @@
+/*
+*	INVENTORY SYSTEM EXTENTION
+*/
+
+// Just by analogy with the TG code. No ideas for what this is.
+/mob/proc/update_inv_vagina()
+	return
+/mob/proc/update_inv_anus()
+	return
+/mob/proc/update_inv_nipples()
+	return
+/mob/proc/update_inv_penis()
+	return
+/// Helper proc for calling all the lewd slot update_inv_ procs.
+/mob/proc/update_inv_lewd()
+	update_inv_vagina()
+	update_inv_anus()
+	update_inv_nipples()
+	update_inv_penis()
```

Comment:
Why do we have all of these if they don't do anything?

Reply:
Base proc, cause a bunch of code makes a few assumptions on mobs, unfortunately. :pain:

Reply:
Or actually it doesn't. Yeet.